pirania-app/files/etc/pirania/governance.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
How is this file defined?
what are each of the fields? looks like a very important one.
also... couldn't it be a /etc/config file (an uci file) instead of a json? in that way it could be backed up by the rest of the system... it would integrate nicely :) (and it is accessed pretty similarly than a json)

Reply:
It's a json because the idea is that it should be shared by shared-state.